app/views/ctc/questions/spouse_filed_prior_tax_year/edit.html.erb

```diff
@@ -1,27 +1,31 @@
-<% content_for :page_title, t("views.ctc.questions.spouse_filed_prior_tax_year.title", prior_tax_year: prior_tax_year, spouse_first_name: current_intake.spouse_first_name) %>
 
-<% content_for :form_card do %>
-  <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
-    <h1 class="h2"><%= content_for(:page_title) %></h1>
-    <p>
-      <%=t("views.ctc.questions.spouse_filed_prior_tax_year.help_text") %>
-    </p>
+<% if current_intake.puerto_rico_filing? %>
+  <%= render 'ctc/questions/spouse_filed_prior_tax_year/pr_edit' %>
```

Comment:
Consider renaming the Puerto Rico file.

@embarnard and I changed a different `_pr_edit.html.erb` to `_puerto_rico_edit.html.erb` a day or two ago. I like the clarity of spelling out `puerto_rico`, so I encourage it here. It doesn't matter a lot, and Em and I haven't done something like presented at Computer Gardening Club about it yet, so not a blocking comment. I do think it'd be nice to be consistent with the spelled-out `_puerto_rico_edit.html.erb`.